common/views/components/Tags/Tags.tsx

```diff
@@ -14,10 +14,12 @@ export type TagType = {
 
 type Props = {
   tags: TagType[];
+  withoutBold?: boolean | null;
 };
 
 const Tags: FunctionComponent<Props> = ({
   tags,
+  withoutBold = false,
```

Comment:
not too fussed, but would it be easier to understand `isBold`?

Reply:
I used without as not all the text inside the text is Bold just some of it

Reply:
Just trying to keep with general naming conventions and to not create confusion with double negative (`withoutBold = true` is is turning without, a negative, into a positive).

`isFirstPartBold` or something that can encapsulate what it is actually doing. I assumed withoutBold was applied to the tag?